src/runtime/exception.cpp

```diff
@@ -85,7 +85,9 @@ static PRIMFN(prim_true) {
 void prim_register_exception(PrimMap &pmap) {
   // These should not be evaluated in const prop, but can be removed
   prim_register(pmap, "stack", prim_stack, type_stack, PRIM_ORDERED);
-  prim_register(pmap, "unreachable", prim_unreachable, type_unreachable, PRIM_ORDERED);
+  prim_register(pmap, "panic", prim_panic, type_panic, PRIM_ORDERED);
```

Comment:
The parameters on one of these should be different right? To do the whole dead code elim allowed/denied?

Reply:
unreachable is treated specially. It's not that ordered makes it special, its that it has the name "unreachable" that makes it special.

Reply:
no wait, crap you're right.